tests/unit/htdocs/Frameworks/art/FunctionsUserTest.php

```
@@ -0,0 +1,137 @@
+
+    // ---------------------------------------------------------------
+    // mod_getIP tests
+    // ---------------------------------------------------------------
+
+    #[Test]
+    public function getIpReturnsValue(): void
+    {
+        $result = @mod_getIP(true);
+        // Should return an IP string or empty
+        $this->assertTrue(is_string($result) || is_long($result));
+    }
```

PHPMD (2.15.0)

[error] Remove error control operator '@' on line 39. (undefined)

(ErrorControlOperator)